src/mongocrypt-ctx-decrypt.c

```diff
@@ -261,7 +261,8 @@ _replace_ciphertext_with_plaintext (void *ctx,
       goto fail;
    }
 
-   plaintext.len = _mongocrypt_calculate_plaintext_len (ciphertext.data.len);
+   plaintext.len =
+      _mongocrypt_calculate_plaintext_len (ciphertext.data.len, status);
```

Comment:
```suggestion
      _mongocrypt_calculate_plaintext_len (ciphertext.data.len, status);
```

Return an error if `_mongocrypt_calculate_plaintext_len` returns 0. Otherwise, an error status may be ignored by the caller.

Reply:
Change incoming.